datarepo-clienttests/src/main/java/scripts/testscripts/DRSLookup.java

```diff
@@ -136,6 +138,19 @@ public void setup(List<TestUserSpecification> testUsers) throws Exception {
             repositoryApi, createSnapshotJobResponse, SnapshotSummaryModel.class);
     logger.info("Successfully created snapshot: {}", snapshotSummaryModel.getName());
 
+    for (TestUserSpecification user : testUsers) {
+      if (!StringUtils.equals(datasetCreator.userEmail, user.userEmail)) {
+        repositoryApi.addSnapshotPolicyMember(
+            snapshotSummaryModel.getId(),
+            "steward",
```

Comment:
do we have constants fo this?

Reply:
sadly no and it's in a separate module so I want to think about that